src/app/owner/pets/dashboard/page.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+
+import { useState } from "react";
+import { mockPets } from "~data/pets/constants";
+import styles from "./page.module.scss";
+import {
+    Card,
+    Image,
+    Text,
+    Button,
+    Group,
+    Stack,
+    Badge,
+    Box,
+    Title,
+} from "@mantine/core";
+
+/**
+ * CREDITS
+ *
+ * Used Claude AI (Anthropic) and ChatGPT-5 to assist with:
+ * - Converting inline styles to SCSS module classes with proper structure
+ * - Restructuring component to match project patterns (removing layout components,
+ *   converting helpers to arrow functions, organizing className references)
+ * - Ensuring Next.js "use client" directive patterns are followed
+ */
+
+const InfoRow = ({ label, value }: { label: string; value: string }) => {
+    return (
+        <div className={styles.info_row}>
+            <span className={styles.info_label}>{label}: </span>
+            <span className={styles.info_value}>{value}</span>
+        </div>
+    );
+};
+
+export default function PetDashboard() {
+    return (
+        <div className={styles.page}>
+            <main>
+                <div className={styles.header}>
+                    <h1 className={styles.title}>My Pets</h1>
+                    <button
+                        className={styles.add_button}
+                        onClick={() =>
+                            (window.location.href = "/owner/pets/create")
+                        }
+                    >
+                        + Add a new pet
+                    </button>
+                </div>
+
+                {/* PET CARD GRID */}
+                <div className={styles.pet_grid}>
+                    {mockPets.map((pet) => (
+                        <div key={pet.id} className={styles.pet_card}>
+                            <div className={styles.pet_image}>
+                                <span className={styles.image_icon}></span>
+                            </div>
+
+                            <div className={styles.pet_content}>
+                                <div className={styles.pet_header}>
+                                    <h3 className={styles.pet_name}>
+                                        {pet.name}
+                                    </h3>
+                                    <button className={styles.edit_badge}>
+                                        EDIT

```

Comment:
```suggestion
                                        Edit

```
Just a nitpick, but since the other buttons use title case, it would be more consistent to make this title case too.